ios/RollbarReactNative.m

```diff
@@ -6,208 +6,356 @@
-  [Rollbar error:message exception:exception data:data context:context];
++ (void)criticalMessage:(NSString*)message {
+  [Rollbar criticalMessage:message];
 }
 
-+ (void)critical:(NSString*)message {
-  [Rollbar critical:message];
++ (void)criticalException:(NSException*)exception {
+  [Rollbar criticalException:exception];
 }
 
-+ (void)critical:(NSString*)message exception:(NSException*)exception {
-  [Rollbar critical:message exception:exception];
++ (void)criticalError:(NSError*)error {
+  [Rollbar criticalError:error];
 }
 
-+ (void)critical:(NSString*)message exception:(NSException*)exception data:(NSDictionary*)data {
-  [Rollbar critical:message exception:exception data:data];
++ (void)criticalMessage:(NSString*)message
+                   data:(nullable NSDictionary<NSString *, id> *)data {
+  [Rollbar criticalMessage:message data:data];
 }
 
-+ (void)critical:(NSString*)message exception:(NSException*)exception data:(NSDictionary*)data context:(NSString*)context {
-  [Rollbar critical:message exception:exception data:data context:context];
++ (void)criticalException:(NSException*)exception
+                     data:(nullable NSDictionary<NSString *, id> *)data {
+  [Rollbar criticalException:exception data:data];
 }
 
-NSString* updateConfiguration(RollbarConfiguration *config, NSDictionary *options) {
-  NSString *accessToken = nil;
++ (void)criticalError:(NSError*)error
+                 data:(nullable NSDictionary<NSString *, id> *)data {
+  [Rollbar criticalError:error data:data];
+}
+
++ (void)criticalMessage:(NSString*)message
+                   data:(nullable NSDictionary<NSString *, id> *)data
+                context:(nullable NSString *)context {
+  [Rollbar criticalMessage:message data:data context:context];
+}
+
++ (void)criticalException:(NSException*)exception
+                     data:(nullable NSDictionary<NSString *, id> *)data
+                  context:(nullable NSString *)context {
+  [Rollbar criticalException:exception data:data context:context];
+}
+
++ (void)criticalError:(NSError*)error
+                 data:(nullable NSDictionary<NSString *, id> *)data
+              context:(nullable NSString *)context {
+  [Rollbar criticalError:error data:data context:context];
+}
+
+RollbarLevel rollbarLevelFromString(NSString *value) {
+    if (value == nil) {
+        return RollbarLevel_Info;
+    } else if ([value caseInsensitiveCompare:@"debug"] == NSOrderedSame) {
+        return RollbarLevel_Debug;
+    } else if ([value caseInsensitiveCompare:@"info"] == NSOrderedSame) {
+        return RollbarLevel_Info;
+    } else if ([value caseInsensitiveCompare:@"warning"] == NSOrderedSame) {
+        return RollbarLevel_Warning;
+    } else if ([value caseInsensitiveCompare:@"error"] == NSOrderedSame) {
+        return RollbarLevel_Error;
+    } else if ([value caseInsensitiveCompare:@"critical"] == NSOrderedSame) {
```

Comment:
Curious: You can't use the static funcs of the exported `RollbarLevelUtil` in `RollbarLevel.h`?

Reply:
I didn't see those. I'll take a look.

Reply:
Will update this in https://github.com/rollbar/rollbar-react-native/pull/169.